src/app/(sidebar)/transaction/fee-bump/page.tsx

```diff
@@ -1,5 +1,200 @@
+          />
+
+          <PositiveIntPicker
+            id="fee"
+            label="Base Fee"
+            value={fee}
+            error={baseErrorMessage}
+            onChange={(e) => {
+              const error = validate.positiveInt(e.target.value);
+              setBaseErrorMessage(error || "");
+              updateBaseFeeBase(e.target.value);
+            }}
+            note={
+              <>
+                The{" "}
+                <SdsLink href="https://developers.stellar.org/docs/learn/fundamentals/fees-resource-limits-metering">
+                  network base fee
+                </SdsLink>{" "}
+                fee is currently set to 100 stroops (0.00001 lumens). Based on
+                current network activity, we suggest setting it to 100 stroops.
+                Final transaction fee is equal to base fee times number of
+                operations in this transaction.
+              </>
+            }
+            infoLink="https://developers.stellar.org/docs/learn/glossary#base-fee"
+          />
+
+          <XdrPicker
+            id="submit-tx-xdr"
+            label="Input a base-64 encoded TransactionEnvelope:"
+            value={innerTxXdr}
+            error={xdrError}
+            onChange={(e) => onXdrChange(e.target.value)}
+            note="Enter a base-64 encoded XDR blob to decode."
+            hasCopyButton
+          />
+        </Box>
+      </Card>
+      <>
+        {feeBumpedTx.xdr ? (
+          <ValidationResponseCard
+            variant="success"
+            title="Success! Transaction Envelope XDR:"
+            response={
+              <Box gap="xs">
+                <TxResponse
+                  label="Network Passphrase:"
+                  value={network.passphrase}
+                />
+                <TxResponse label="XDR:" value={feeBumpedTx.xdr} />
+              </Box>
+            }
+            note={<></>}
+            footerLeftEl={
+              <>
+                <Button
+                  size="md"
+                  variant="secondary"
+                  onClick={() => {
+                    updateSignImportXdr(feeBumpedTx.xdr);
+                    updateSignActiveView("overview");
+
+                    router.push("/transaction/sign");
+                  }}
+                >
+                  Sign in Transaction Signer
+                </Button>
+                <Button
+                  size="md"
+                  variant="tertiary"
+                  onClick={() => {
+                    alert("TODO: handle view in xdr flow");
+                  }}
+                >
+                  View in XDR viewer
+                </Button>
+              </>
+            }
+          />
+        ) : null}
+      </>
+      <>
+        {feeBumpedTx.errors.length && innerTxXdr ? (
+          <ValidationResponseCard
+            variant="error"
+            title="Transaction Sign Error:"
+            response={feeBumpedTx.errors}
+          />
+        ) : null}{" "}
```

Comment:
Do we need the trailing space here?

```suggestion
        ) : null}
```